Automattic/sensei
# Log when extensions are installed or updated on the Extensions page

### Changes proposed in this Pull Request
Logs when extensions are installed or updated on the Extensions page. Also moves the event logging script to the shared folder.

I may add `success` events for these, but if so it will be after the installing and updating PRs are merged.

### Testing instructions

- Enable usage tracking.
- On the Extensions page, click the _Update_ button on a card.
- Ensure `sensei_extensions_update` is logged and `slug` is set appropriately.
- Click the _Update_ button in the notification.
- Ensure `sensei_extensions_update` is logged and `slug` is set appropriately.
- Install another outdated Sensei extension and click the _Update all_ button in the notification.
- Ensure one `sensei_extensions_update` event is logged per extension and that `slug` is set appropriately.
- Click the _Install_ button on a card.
- Ensure `sensei_extensions_install` is logged and `slug` is set appropriately.
